README.md

```diff
@@ -176,3 +176,17 @@ command.
 ### Still having issues?
 
 Check out our [troubleshooting](troubleshooting.md) doc.
+
+## Deployment
+
+In order to get your changes deployed there might be several merges/upgrades to sort out.  For example, if you update a banner, requiring changes in `cp-translations`, `cp-zen-frontend`, and `cp-zen-platform`, you need to:
+
+1. Merge `cp-translations` changes.  Re-pull your master branch to get the auto-incremented version.
+2. Update the version of `cp-translations` in packages.json for `cp-zen-frontend`, and re-install yarn dependencies, and get them merged to master.  Re-puu your master branch to get the auto-incremented version number.
+3. Finally update the version of `cp-zen-frontend` in `cp-zen-platform`, re-install yarn dependencies, and then get those changes merged to master.
```

Comment:
Need to update the version of cp-translations in cp-zen-platform maybe too?

Reply:
Does that not get pulled in automatically?..?